### PR TITLE
Allow authentication with bearer token on alerts and alertmanager commands

### PR DESCRIPTION
**Pull request by javad-hajiani**
Cortextool client only supports authToken on rules command. This PR adds authToken argument to alerts and alertmanager commands.

### CODE REVIEW

**Review by gouthamve — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -36,12 +36,13 @@ Config commands interact with the Cortex api and read/create/update/delete user
...
-| CORTEX_TENANT_ID | `id`      | The tenant ID of the Cortex instance to interact with.                                                        |
+| Env Variables     | Flag       | Description                                                                                                   |
+| ----------------- | ---------  | ------------------------------------------------------------------------------------------------------------- |
+| CORTEX_ADDRESS    | `address`  | Address of the API of the desired Cortex cluster.                                                              |
+| CORTEX_API_USER   | `user`     | In cases where the Cortex API is set behind a basic auth gateway, a user can be set as a basic auth user. If empty and CORTEX_API_KEY is set, CORTEX_TENANT_ID will be used instead. |
+| CORTEX_API_KEY    | `key`      | In cases where the Cortex API is set behind a basic auth gateway, a key can be set as a basic auth password. |
+| CORTEX_AUTH_TOKEN | `authToken`| In case where the Cortex API is set behind gateway authenticating by bearer token, a token can be set as a bearer token header. |
```
Comment (colega) [inline README.md:44]: Nit: just to be consistent.

```suggestion
| CORTEX_AUTH_TOKEN | `authToken`| In cases where the Cortex API is set behind gateway authenticating by bearer token, a token can be set as a bearer token header. |
```

Comment (javad-hajiani) [reply README.md]: Done!

**Review by colega — APPROVED** [review]

**Comment by colega** [comment]
LGTM, thank you!

**Pull request merged**